.github/workflows/containers.yml

```diff
@@ -17,23 +17,34 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        container: [ functional, integration, unit ]
+        include:
+          - directory: docker/network
+            prefix: ovirt/vdsm-network-tests
+            container: [ functional, integration, unit ]
+          - directory: docker
+            prefix: ovirt/vdsm-test
+            container: [ centos-8, centos-9 ]
     steps:
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: |
           sudo apt update
           sudo apt install podman
       - name: Build container images
-        working-directory: docker/network
+        working-directory: ${{ matrix.directory }}
         run: make ${{ matrix.container }}
+      - id: tag-selector
+        with:
+          cond: ${{ endsWith(matrix.directory, 'network') }}
+          if_true: alma-9 centos-8 centos-9
+          if_false: ${{ matrix.container }}
```

Comment:
I think this will give us something else than we're currently using. IIUC, the resulting images for `docker/Makefile` will have names/tags like this one: `ovirt/vdsm-test-centos-8:centos-8`. Currently we're building and pushing `ovirt/vdsm-test-centos-8`, which resolves to `ovirt/vdsm-test-centos-8:latest` implicitly.

This actually shows the discrepancy between the network and regular test images and the ugliness it causes. I'd rather see the names unified first and get rid of the `tag-selector` step completely. So if @almusil is OK with that, I'd rename the network images from i.e. `ovirt/vdsm-network-tests-functional:el8stream` to `ovirt/vdsm-network-tests-functional-centos-8:latest`. Also, I don't see a reason why we shouldn't use alma 9 for regular tests too.

Reply:
Ah, I see. I understand the tags better now. Then, I could explicitly set `latest` here if false, but unifying strategies sounds like the best approach, completely agree. It would also simplify this update.

Reply:
Using the OS in name has a huge disadvantage. It effectively multiplies how many containers are needed. It also does not inline with naming for e.g. centos. What is blocking us to do it the other way around for vdms-test? I mean if that's hard requirement to change the network containers go ahead just keep in mind that it means triple the amount of containers currently.

Reply:
I agree with your point. I don't think there's anything blocking us from going the other direction. I do see though that we also have imageio images that follow the other convention, so I've picked the approach that needs less work :) Maybe we'll convince @nirs to move to the other convention too

Reply:
I don’t think it makes any difference, you need n container files and n builds for running n ci builds.

Use what is easier to use an manage.